SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/AppCenter.cs

```diff
@@ -425,6 +433,26 @@ private void StartService(IAppCenterService service)
             AppCenterLog.Info(AppCenterLog.LogTag, $"'{service.GetType().Name}' service started.");
         }
 
+        private async Task SendStartServiceLog(IEnumerable<string> serviceNames)
+        {
+            if (InstanceEnabled)
```

Comment:
This check is not atomic. On Android we run commands in a loop so there is only 1 thread involved, but in .NET you have multiple threads involved when using tasks.